src/scene-entry-manager.js

```diff
@@ -82,9 +83,16 @@ export default class SceneEntryManager {
     cursor.enable();
     cursor.setCursorVisibility(true);
 
-    this.hubChannel.sendEntryEvent().then(() => {
-      this.store.update({ activity: { lastEnteredAt: new Date().toISOString() } });
-    });
+    // Delay sending entry event telemetry until VR display is presenting.
```

Comment:
Why are we doing this?

Reply:
see: https://github.com/mozilla/hubs/pull/618